interop_binaries/src/bin/janus_interop_client.rs

```diff
@@ -113,7 +113,7 @@ async fn handle_upload(
     let vdaf_instance = request.vdaf.clone().into();
     match vdaf_instance {
         VdafInstance::Prio3Count {} => {
-            let measurement = parse_primitive_measurement::<u64>(request.measurement.clone())?;
+            let measurement = parse_primitive_measurement::<bool>(request.measurement.clone())?;
```

Comment:
Note that this will change the interop test API's effective JSON schema. I think we should leave this as a number, and convert the measurement before we pass it to the VDAF. Note that the libprio-rs test vector tests had to do something similar for this change.

Reply:
Another option would be to publish a new version of the interop test draft to work with VDAF-08. However, given that VDAF itself still indicates that the measurement type is numeric rather than boolean, I think it is likely better to consider `libprio-rs`' use of bool as an implementation choice and _not_ move it out to the other drafts.

That is, I agree with David that this portion of the change should be reverted.

Reply:
Issue for interop draft changes: https://github.com/divergentdave/draft-dcook-ppm-dap-interop-test-design/issues/43

And I made a note to sync up with those in #2389.